pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -754,19 +753,16 @@ func removeTerminatedContainers(containers []*runtimeapi.Container) []*runtimeap
 	result := make([]*runtimeapi.Container, 0)
 	for _, refs := range containerMap {
 		if len(refs) == 1 {
```

Comment:
This entire if block can be removed, correct?

Reply:
Ah yes it can. Updating.